src/Barman.java

```diff
@@ -0,0 +1,32 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Barman {
+    public Drink createDrink() {
+        Scanner scanner = new Scanner(System.in);
+        List<Ingredient> ingredients = new ArrayList<>();
+        for (int i = 0; i < 3; i++) {
+            Ingredient ingredient = loadIngredient(scanner);
+            ingredients.add(ingredient);
+        }
+        return new Drink(ingredients);
+    }
+
+    private Ingredient loadIngredient(Scanner scanner) {
+        System.out.println("Podaj nazwę składnika: ");
+        String name = scanner.nextLine();
+        System.out.println("Podaj ilość składnika w mililitrach: ");
+        int amount = scanner.nextInt();
+        scanner.nextLine();
+        return new Ingredient(name, amount);
+    }
+
+    public void printDrink(Drink drink) {
```

Comment:
Why do you have printring drink method in Barman class if it is directly related to a Drink? Wouldn't it be cool to have it in Drink class?